tests/test_hdate.py

```diff
@@ -397,6 +397,16 @@ def test_get_omer_day(self, execution_number, rand_hdate):
             rand_hdate.hdate = HebrewDate(rand_hdate.hdate.year, 9, day)
             assert rand_hdate.omer_day == day + 44
 
+    def test_daf_yomi(self):
+        # Random test date
+        myhdate = HDate(gdate=datetime.date(2014, 4, 28), hebrew=False)
+        assert myhdate.daf_yomi == ("Beitzah", 29)
```

Comment:
Please add a test for hebrew as well

Reply:
Fixed in a6d7f7bce7455c7c608e35e2ebebf52238082df3